tests/mcp/test_mcp_http.py

```
@@ -0,0 +1,11 @@
+from mcp.client.streamable_http import streamablehttp_client
+from mcp.client.session import ClientSession
+
+async def main():
+    async with streamablehttp_client("http://0.0.0.0:11234/mcp/http") as (r,s,_):
+        async with ClientSession(r,s) as sess:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)